### PR TITLE
Refactoring/update identifier data type for selected entities

### PR DESCRIPTION
**Pull request by h4wk13**

### CODE REVIEW

File: OutOfSchool/OutOfSchool.DataAccess/Models/Teacher.cs [hunk 1]
```diff
@@ -1,44 +1,23 @@
...
-        [MaxLength(300)]
-        public string Description { get; set; } = string.Empty;
+        public string Description { get; set; }
 
-        [MaxLength(256)]
-        public string Image { get; set; } = string.Empty;
+        public string Image { get; set; }
 
-        [Required]
         public long WorkshopId { get; set; }
```
Comment (OlhaHoliak) [inline OutOfSchool/OutOfSchool.DataAccess/Models/Teacher.cs:21]: Guid?

File: OutOfSchool/OutOfSchool.WebApi/Services/ChatMessageService.cs [hunk 2]
```diff
@@ -51,23 +53,21 @@ public async Task<ChatMessageDto> Create(ChatMessageDto chatMessageDto)
...
 
-                if (!(chatMessage is null))
-                {
-                    await repository.Delete(chatMessage).ConfigureAwait(false);
-                }
-                else
+                if (chatMessage is null)
                 {
-                    throw new ArgumentOutOfRangeException(nameof(id), $"ChatMessage with id:{id} was not found in the system.");
+                    return;
```
Comment (OlhaHoliak) [inline OutOfSchool/OutOfSchool.WebApi/Services/ChatMessageService.cs:67]: should not we log that chat message was null?

**Review by VadymLevkovskyi — CHANGES_REQUESTED** [review]

File: OutOfSchool/OutOfSchool.ElasticsearchData/Models/WorkshopES.cs [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
+using System.Collections.Generic;
+
 using Nest;
 
 namespace OutOfSchool.ElasticsearchData.Models
 {
     public class WorkshopES
     {
-        public long Id { get; set; }
+        public Guid Id { get; set; }
```
[the remark this answers is not in the document]
Comment (h4wk13) [reply OutOfSchool/OutOfSchool.ElasticsearchData/Models/WorkshopES.cs:10]: Haven't seen such attributes in other ES models.
Should it be separate task ?

File: OutOfSchool/Tests/OutOfSchool.Tests.Common/TestDataGenerators/ApplicationDTOsGenerator.cs [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+namespace OutOfSchool.Tests.Common.TestDataGenerators
+{
+    /// <summary>
+    /// Contains methods to generate fake <see cref="ApplicationDto"/> objects.
+    /// </summary>
+    public static class ApplicationDTOsGenerator
+    {
+        private static readonly TimeSpan timeShift = TimeSpan.FromDays(5);
+
+        private static readonly Faker<ApplicationDto> faker = new Faker<ApplicationDto>()
```
[the remark this answers is not in the document]
Comment (h4wk13) [reply OutOfSchool/Tests/OutOfSchool.Tests.Common/TestDataGenerators/ApplicationDTOsGenerator.cs:19]: Do we really need comments on private fields ?

**Review by VadymLevkovskyi — APPROVED** [review]

**Pull request merged**